product_docs/docs/pgd/5/quick_start.mdx

```diff
@@ -1,14 +1,15 @@
 ---
-title: "Example: Deploying EDB Postgres Distributed"
-navTitle: "Example: Deploying EDB Postgres Distributed"
+title: "Deploying an EDB Postgres Distributed example cluster on AWS"
+navTitle: "Deploying EDB Postgres Distributed"
 description: >
   A quick demonstration of deploying a PGD architecture using TPA on Amazon EC2
 redirects:
   - /pgd/latest/deployments/tpaexec/quick_start/
+  - /pgd/latest/tpa/quick_start/
 ---
 
 
-The following steps setup EDB Postgres Distributed with an Always On Single Location
+The following steps set up EDB Postgres Distributed with an Always On Single Location
```

Comment:
I think we need some prerequisite type info that describes what was done before the example starts - something like: TPA is installed in your AWS environment

Question: will we be replacing this topic with a new topic using docker or will they be siblings?